test/hotspot/jtreg/TEST.groups

```diff
@@ -176,6 +177,11 @@ tier1_compiler_3 = \
 tier1_compiler_not_xcomp = \
   compiler/profiling
 
+tier2_compiler =
+
+tier3_compiler = \
+  compiler/arraycopy/stress
```

Comment:
Can you introduce separate group for this? For example `hotspot_arraycopy_stress` and use it here.
I am fine with introduced `tier2|3_compiler` groups but it will help us in Oracle to have separate group for `arraycopy` so we can schedule its testing on proper machines.

Reply:
Yes, we can. Actually, working on #6622, I realized these test groups would be introduced anyway. So these new arraycopy tests should probably go to `hotspot_slow_compiler` group, along with other `stress` tests. This would hook arraycopy tests into `hotspot:tier3` automatically. Tell me if you still want a completely separate test group, or `hotspot_slow_compiler` is enough for current Oracle testing infra.

Reply:
Please,  create separate test group and add it to `hotspot_slow_compiler`. We would not need to change infra settings if more testing is added to this new group later.

Reply:
Done in new commit.